docs/source/concept_models/vivarium_iv_iron/concept_model.rst

```diff
@@ -208,6 +215,59 @@ For GBD outcomes that do not have estimates available for the locations of inter
 
 Details on how to calculate weighted averages for specific simulation parameters are shown on the :ref:`women of reproductive age simulation <2019_concept_model_vivarium_iv_iron_maternal_sim>` and the :ref:`children under five simulation <2019_concept_model_vivarium_iv_iron_child_sim>` pages.
 
+Expanding to additional locations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As new locations are added to this model, there are several data values that need to be updated in the artifact in addition to standard GBD data updates. These calculations are performed in the `vivarium_research_iv_iron repository <https://github.com/ihmeuw/vivarium_research_iv_iron>`_  and include:
+
+- Weights used in calculation of the below values
+  
+  - `Calculation performed in this notebook <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/parameter_aggregation/Generate%20weights.ipynb>`_
+
+- Hemoglobin validation targets
+ 
+  - `Done for prevalence here <https://github.com/ihmeuw/vivarium_research_iv_iron/blob/main/hgb_validation_targets/Generate%20validation%20targets.ipynb>`_. Note that the anemia prevalence .csv file is too large to store in git, so it will have to be generated locally. Remeber to delete it before commiting changes in order to avoid annoying git issues.
```

Comment:
You might want to add the filepath to the `.gitignore` file in the repo ([see documentation](https://git-scm.com/docs/gitignore)) to make sure it is never accidentally committed!

Reply:
Good thinking :) will do, thanks